templates/include/telemetry/coreCommon.2.schema.json

```diff
@@ -152,4 +152,16 @@
     "type": "string",
     "description": "Added in bug 1612935."
```

Comment:
Could you also add a note that these fields are deprecated and should not contain any data?